src/util.js

```diff
@@ -1346,16 +1349,18 @@ async function commitCaptchaTokenWithSelector(
       2
     );
 
-    const id = await captchaSolver.send({
+    captchaId = await captchaSolver.send({
       method: "base64",
       body: base64,
       max_len: captchaLength,
       min_len: captchaLength,
     });
 
-    global.currentCaptchaId = id;
+    global.currentCaptchaId = captchaId;
 
-    const token = await captchaSolver.get(id);
+    const token = await captchaSolver.get(captchaId);
+    await captchaSolver.reportGood(captchaId);
```

Comment:
We report good captcha immediately after it is solved. Which is not extremely wrong given captcha is 99% correct in our expereince.